executor/src/witgen/machines/fixed_lookup_machine.rs

```diff
@@ -265,6 +307,14 @@ impl<'a, T: FieldElement> FixedLookup<'a, T> {
             }
         };
 
+        // Update the multiplicities
+        self.multiplicities
```

Comment:
This also stores the value if we don't have/need a multiplicities column, right?

Reply:
Please only do this if we have a multiplicities column.